pages/administratorServer/administratorServer.ejs

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <%- include('../partials/head'); %>
+  </head>
+  <body>
+    <script>
+      $(function () {
+          $('[data-toggle="popover"]').popover({sanitize: false})
+      });
+    </script>
+    <%- include('../partials/navbar', {navPage: 'admin', navSubPage: 'server'}); %>
+    <div id="content" class="container-fluid">
+
+      <!-------------------------------------------------------------------------------->
+      <!-------------------------------------------------------------------------------->
+      <!-------------------------------------------------------------------------------->
+      <!-- Config ---------------------------------------------------------------------->
```

Comment:
Could delete these comments now.

Reply:
Turns out this whole page was unused; I removed it in cdf33e0.